### ISSUE TITLE
Chat windows

### ISSUE THREAD

**Issue by Oonna** [failcoder/Supervillain_UI#140]
Since the new 6.2 patch when I'm playing everything in my chat windows disappears. I have to click back on the chat tabs to get the text to reappear. Sometimes it stays up for a few mins other times it's right after I type something and hit enter. Is anyone else experiencing this? Is there a setting I need to change?

**Comment by OrbitStorm** [comment]
This has been happening to my wife and I as well. We just assumed it was some new [terrible] feature but I hadn't gotten around to searching for it. Hopefully this is just a bug that can be corrected going forward. :)

**Comment by penguinsane** [comment]
I have noticed this happening only after I do a pet battle. Are either of you playing pet battles?

**Comment by OrbitStorm** [comment]
Unfortunately, it seems to happen for both my wife and I rather intermittently. Pet battles definitely does trigger it but I think there might be other events/actions that are triggering the vanishing act too.

**Comment by Littlekong** [comment]
I have exactly the same issue. It's happened when I am in combat or in group/raid.
It's very annoying. I Think it's a bug because they are nothing more on the options menu for the chat settings.

**Comment by Oonna** [comment]
It doesn't matter what I'm doing. I could just be hanging out in my garrison chatting with my guild and send a message and right after I send it my chat disappears.

**Comment by Waynos** [comment]
Yeah, I am having this problem too.  Nothing inparticular triggers the loss of the text, just happens randomly.  Finding it very annoying tbh.  Noticed in the update logs that issues with chat have had minor fixes, but not for this problem.  I have looked through all the interface settings and also the blizzard interface settings to see if there is something enabling "Fade Chat" but cant find anything.  Are the UI developers aware of this problem and trying to fix it?  Great UI and really enjoying using it, but this is something that is starting to irritate me as I keep missing messages when in raids etc.

**Comment by Waynos** [comment]
Seemed like the chat windows problem was fixed up until recently (maybe something to do with installing windows 10?).  I have not changed my chat settings but now, I can see the General window and everything in there fine, I can see the loot window and that appears fine, however, custom windows I have (Guild, PM's and Group) have two problems.  First, the name of the window is missing from the actual tab, until I click on that tab group.  Secondly, nothing appears in the chat window for any of these three groups.  I have double checked to make sure that I have got all the setttings right as to what shows in each window.  I have tried reclogging, resetting the UI to default, deleting and reinstalling the UI via Curse.  Nothing works.  I have recently stopped using ElvUI altogether now in favour of this UI.  Its a great UI but this is really an annoying problems for me, especially as I like to keep my chat windows organised by what I am doing, ie Guild chat, Raid, PM's etc.  Hope a fix is coming soon, I have faith :D  Thanks for listening to my rant :P
